### PR TITLE
Development

### PR DESCRIPTION
**Pull request by epinczinger**
The project was built mainly with bootstrap for responsive design. The aim was to copy the original [NewsWeek](https://www.newsweek.com/) page. It has a small size view, a medium and a large one.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @epinczinger, awesome job with this project :+1: UI looking great, great commit messages, README professional :slightly_smiling_face: However, in order to get our project approved, we need to address the comments below:
- Kindly enable stickler by visiting stickler-ci.com to check for style errors/warnings
- ```Mobile device 605px```, let us fix the positioning by comparing our site to the source
``` clone ```
605px
``` source ```
605pxS
``` US, Sports, Culture, Tech and Science, Business, World  ``` sections, let us align the elements
``` clone ```
sprtheight
``` source ```
sprtHeight1
- Let add a little margin around the ```Editors Pick``` so it looks like the source
``` clone ```
edpicks
``` source ```
edpick1
- Let us add a gray background to the See All Features and increase the height of our images a little and the date
``` clone ```
seeFeatures
``` source ```
seeFeatures1

Kindly fix these comments and send another code review :smiley: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by epinczinger** [comment]
Hi, @addod19 thanks for your review! I've fixed the changes you pointed. I'm submitting for another check.
Kind regards

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @epinczinger ! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Please take a look at your footer once again in the `605 px` screen size. The layout is still in need of improvement as was requested by the previous TSE. 
Your clone 👇 
Screen Shot 2020-06-11 at 2 30 55 AM
The original page 👇 
Screen Shot 2020-06-11 at 2 30 46 AM

- [x] As was requested by the previous TSE, the US, Sports, Culture, Tech and Science, Business, and World sections should be aligned. Please compare your clone with the original page and fix this.
Your clone 👇 
Screen Shot 2020-06-11 at 2 33 12 AM
The original page 👇 
Screen Shot 2020-06-11 at 2 32 51 AM

- [x] The date on the "see all features" should be on the magazine cover, not below it. Additionally, this section is not aligned properly. Please compare your clone with the original page and fix this. 
Your clone 👇 
Screen Shot 2020-06-11 at 2 35 53 AM
The original page 👇 
Screen Shot 2020-06-11 at 2 36 10 AM

- [x] In the "in the magazine section" on the medium screen size `(768 px)` the title of the magazines should be in one line. 
Your clone 👇 
Screen Shot 2020-06-11 at 2 35 18 AM
The original page 👇 
Screen Shot 2020-06-11 at 2 35 38 AM





****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by epinczinger** [comment]
Hi! I've fixed the changes required. I think the live demo link you opened wasn't correctly updated because I had fixed the alignments and the all features date before.
I've tried to refresh it so I hope you can see it properly now. Sorry for the inconvenient.

Kind Regards

(I'm not sure why sometimes on the live demo link some images are not displayed properly)

**Comment by tirthajyoti-ghosh** [comment]
# Pull Request under review by T. Ghosh (TSE)

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Impressive project @epinczinger ✨👏, awesome README btw ✨

### However, your project still need some changes in order to get it approved.

- The `navbar` needs to have spaces on either side.

   Right now 👇

   Annotation 2020-06-11 233852

   Original 👇

   Annotation 2020-06-11 234059

- Increase the width of the `top story` section and decrease the width `the debate/ opinion` section
  
  Annotation 2020-06-11 234239

- Increase the width of the image in `sign-up`
  
  Annotation 2020-06-11 234551

- Bottom-left tags are missing in `editor's picks` section

   Right now 👇

   Annotation 2020-06-11 234740

   Original 👇

   Annotation 2020-06-11 234908

-  Entire `subscribe/newsletter` section is missing

   Right now 👇

   Annotation 2020-06-11 235046

   Original 👇

   Annotation 2020-06-11 235058

- The footer must also have space on either side.

   Right now 👇
   
   Annotation 2020-06-11 235209

   Original 👇
  
   Annotation 2020-06-11 235220

I have also left some comments below, be sure to check them.

File: assets/style.css [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+}
+
+main {
+  max-width: 1310px;
+  margin: 0 auto;
+}
+
+a {
+  color: inherit;
+  text-decoration: none !important;
```
Comment (tirthajyoti-ghosh) [inline assets/style.css:17]: - Remove `!important`.

File: assets/style.css [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  font: 700 17px/21px 'Roboto Condensed', sans-serif;
+}
+
+.n-magazine h3 .editor-pick h3 {
+  font: 700 19px/26px 'Roboto Condensed', sans-serif;
+  margin: 6px 0 0 5px;
+}
+
+img[src="assets/images/n.jpeg"] {
+  width: 29px !important;
```
Comment (tirthajyoti-ghosh) [inline assets/style.css]: - Remove `!important`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1057 @@
...
+  <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" data-auto-replace-svg="nest"></script>
+
+</head>
+
+<body>
+  <header>
+
+    <!-- PRINCIPAL NAVBAR  -->
+
+    <nav class="navbar sticky-top py-lg-3" style="background-color: red;">
```
Comment (tirthajyoti-ghosh) [inline index.html]: - Remove in-line styles, here and in all other places. It's a CSS bad practice.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply these changes and request for another code review.

Happy coding 👨‍💻

**Comment by epinczinger** [comment]
Hi! Changes have been done. Please check the current version because the live link seems to have a delay. 
Kind regards.

**Comment by peterrobert** [comment]
### PULL REQUEST UNDER REVIEW BY PETER ROBERT (TSE)

**Review by peterrobert — APPROVED** [review]
### APPROVED :+1: :+1: :100: 
**GREAT WORK IMPROVING ON ALL THE REQUIRED CHANGES FROM THE PREVIOUS TSE** :+1: 
Your project looks great :+1:  

**[OPTIONAL]** 
You can make the search bar look like the original. Yours :arrow_down: 
search
original :arrow_down: 
or

**ONCE YOU MAKE THAT CHANGE KINDLY MERGE THE BRANCH AND SUBMIT THE COMPLETION FORM** :1st_place_medal: 

### HAPPY CODING :computer:

**Comment by epinczinger** [comment]
Thanks! I might have changed the colour when modified the CSS, but I'll fix it. Regards

**Pull request merged**